Userland/Libraries/LibJS/Runtime/ModuleNamespaceObject.cpp

```diff
@@ -0,0 +1,197 @@
+/*
+ * Copyright (c) 2022, David Tuin <davidot@serenityos.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <AK/QuickSort.h>
+#include <LibJS/Runtime/GlobalObject.h>
+#include <LibJS/Runtime/ModuleNamespaceObject.h>
+
+namespace JS {
+
+ModuleNamespaceObject::ModuleNamespaceObject(GlobalObject& global_object, Module* module, Vector<FlyString> exports)
+    : Object(*global_object.object_prototype())
+    , m_module(module)
+    , m_exports(move(exports))
+{
+    // Note: We just perform step 6 of 10.4.6.12 ModuleNamespaceCreate ( module, exports ), https://tc39.es/ecma262/#sec-modulenamespacecreate
+    // 6. Let sortedExports be a List whose elements are the elements of exports ordered as if an Array of the same values had been sorted using %Array.prototype.sort% using undefined as comparefn.
+    quick_sort(m_exports, [&](FlyString const& lhs, FlyString const& rhs) {
+        return lhs.view() < rhs.view();
+    });
+}
+
+void ModuleNamespaceObject::initialize(GlobalObject& global_object)
+{
+    Object::initialize(global_object);
+    define_direct_property(*vm().well_known_symbol_to_string_tag(), js_string(vm(), "Module"sv), PropertyAttributes(0));
```

Comment:
```suggestion

    // 28.3.1 @@toStringTag, https://tc39.es/ecma262/#sec-@@tostringtag
    define_direct_property(*vm().well_known_symbol_to_string_tag(), js_string(vm(), "Module"sv), 0);
```

(yes, that's the actual title and URL :thonk:)

Reply:
The only true `@@toStringTag`? That's weird